### PR TITLE
Integration Mpay24Seamless PaymentProvider

### PR DESCRIPTION
**Pull request by andreas-gruenwald**
Additional Payment Provider for the Ecommerce framework of Pimcore.

### CODE REVIEW

File: pimcore/lib/Pimcore/Bundle/EcommerceFrameworkBundle/Resources/config/payment_manager.yml [hunk 1]
```diff
@@ -31,6 +31,11 @@ services:
             $options: []
             $formFactory: '@form.factory'
 
+    Pimcore\Bundle\EcommerceFrameworkBundle\PaymentManager\Payment\;Mpay24Seamless:
```
Comment (fashxp) [inline pimcore/lib/Pimcore/Bundle/EcommerceFrameworkBundle/Resources/config/payment_manager.yml]: is this semicolon a mistake or on purpose? `;Mpay24Seamless`

Comment (andreas-gruenwald) [reply pimcore/lib/Pimcore/Bundle/EcommerceFrameworkBundle/Resources/config/payment_manager.yml]: It's a mistake. It should look the same way as it is for other payment providers:

```   
 Pimcore\Bundle\EcommerceFrameworkBundle\PaymentManager\Payment\WirecardSeamless:
        arguments:
            $options: []
            $templatingEngine: '@templating.engine.delegating'
            $session: '@session'
```

Comment (fashxp) [reply pimcore/lib/Pimcore/Bundle/EcommerceFrameworkBundle/Resources/config/payment_manager.yml]: could you change it? :)

**Comment by fashxp** [comment]
could you please make sure that the tests run through. thanks.

**Comment by fashxp** [comment]
done necessary changes.

**Pull request merged**